gradle-android-command-plugin/src/main/groovy/com/novoda/gradle/command/Start.groovy

```diff
@@ -0,0 +1,4 @@
+package com.novoda.gradle.command
+
+class Start extends Run {
```

Comment:
Might make sense to have a `start` alias for run. Should also be wire by default, if we want to go with it.